runtimes/python-3.10/README.md

```diff
@@ -18,13 +18,13 @@ printf "import random\n\ndef main(req, res):\n    return res.json({'n': random.r
 2. Build the code:
 
 ```bash
-docker run --rm --interactive --tty --volume $PWD:/usr/code open-runtimes/python:3.10 sh /usr/local/src/build.sh
+docker run -e ENTRYPOINT_NAME=main.py --rm --interactive --tty --volume $PWD:/usr/code openruntimes/python:3.10 sh /usr/local/src/build.sh
```

Comment:
Why do we need a `ENTRYPOINT_NAME` 

1. build.sh does not seem to be using this env variable anywhere
2. Didn't we rename it to `INTERNAL_RUNTIME_ENTRYPOINT`  ??

Please check this in the other files as well.